test/integ/matMul/src/matMul.cpp

```diff
@@ -31,27 +31,21 @@ class MatMulKernel
     //! \param m The height of the A matrix.
     //! \param n The width of the A and height of the B matrix.
     //! \param k The width of the B matrix.
-    //! \param A The pointer to the matrix A data.
-    //! \param lda The pitch of the A matrix in elements.
-    //! \param B The pointer to the matrix B data.
-    //! \param ldb The pitch of the B matrix in elements.
-    //! \param C The pointer to the matrix C data.
-    //! \param ldc The pitch of the C matrix in elements.
+    //! \param A The accessor to the matrix A data.
+    //! \param B The accessor to the matrix B data.
+    //! \param C The accessor to the matrix C data.
     ALPAKA_NO_HOST_ACC_WARNING
-    template<typename TAcc, typename TElem, typename TIndex>
+    template<typename TAcc, typename TMemoryHandle, typename TElem, typename TIndex>
     ALPAKA_FN_ACC auto operator()(
         TAcc const& acc,
         TIndex const& m,
         TIndex const& n,
         TIndex const& k,
         TElem const& alpha,
-        TElem const* const A,
-        TIndex const& lda,
-        TElem const* const B,
-        TIndex const& ldb,
+        alpaka::Accessor<TMemoryHandle, TElem, TIndex, 2, alpaka::ReadAccess> const A,
+        alpaka::Accessor<TMemoryHandle, TElem, TIndex, 2, alpaka::ReadAccess> const B,
         TElem const& beta,
-        TElem* const C,
-        TIndex const& ldc) const -> void
+        alpaka::Accessor<TMemoryHandle, TElem, TIndex, 2, alpaka::ReadWriteAccess> const C) const -> void
```

Comment:
`BufferAccessor` is also required here.

Reply:
I don't think `BufferAccessor` is required, just having multiple `TMemoryHandle`.